### PR TITLE
✨(frontend)  share modal

### PR DESCRIPTION
**Pull request by PanchoutNathan**
Implement the new share modal ui

### CODE REVIEW

File: src/frontend/apps/impress/src/features/docs/doc-share/component/DocShareMemberItem.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            <DocRoleDropdown
+              currentRole={access.role}
+              onSelectRole={onUpdate}
+              canUpdate={doc.abilities.accesses_manage}
+              isLastOwner={isLastOwner}
+              isOtherOwner={!!isOtherOwner}
+            />
+
+            {isDesktop && doc.abilities.accesses_manage && (
+              <DropdownMenu options={moreActions}>
```
Comment (AntoLC) [inline src/frontend/apps/impress/src/features/docs/doc-share/component/DocShareMemberItem.tsx:90]: Only on Firefox apparently, the dropdown don't open everytime.

https://github.com/user-attachments/assets/664ef2ce-b529-4dd2-8735-8bfe1d3bf2d7

**Review by AntoLC — COMMENTED** [review]
Good job ^^ It is nice.

I think you can finish to clear the feature `members`.

File: src/frontend/apps/impress/src/features/docs/doc-share/component/DocShareMemberItem.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                <IconOptions
+                  data-testid="doc-share-member-more-actions"
+                  $variation="600"
+                />
+              </DropdownMenu>
```
[the remark this answers is not in the document]
Comment (rl-83) [reply src/frontend/apps/impress/src/features/docs/doc-share/component/DocShareMemberItem.tsx:92]: It's a feature:
1) Displaying a trash can make users uneasy by suggesting—even falsely—that they might accidentally delete someone's account with a single click.
2) Adds an extra step to ensure the user actually wants to delete someone
3) In the "more" button, it is possible to add future features
4) Displaying many trash cans is ugly 😆

**Review by AntoLC — APPROVED** [review]
Bravo ^^

**Pull request merged**